pywren_ibm_cloud/executor.py

```diff
@@ -104,6 +105,9 @@ def __init__(self, config=None, runtime=None, runtime_memory=None, compute_backe
         self.cleaned_jobs = set()
         self.last_call = None
 
+        if self.auto_dismantle:
+            atexit.register(self.dismantle)
```

Comment:
@JosepSampe Nice job! One concern. Is atexit promised to be called before invoker object garbage collected?

Reply:
For the moment I deleted this from executor to avoid conflicts with master branch. 

@kpavel Now with your amazing contrib in #350, I'm wondering if it is necessary any auto_dismantle logic in the executor. In my opinion, maybe it is better to always let the VMs be automatically self-stopped after 5 minutes idle instead of having to stop them after each `exec.get_result()` or when the script finishes its execution with the `atexit` module.